spec/syntax.md

```diff
@@ -406,7 +406,7 @@ and `\` (which starts an escape sequence).
 ```ebnf
 Text ::= (TextChar | TextEscape)+ /* ws: explicit */
 TextChar ::= AnyChar - ('[' | ']' | '{' | '}' | Esc)
-AnyChar ::= .
+AnyChar ::= [#x0-#x10FFFF]
```

Comment:
This change should also be reflected in the `message.ebnf` file.

Reply:
Done.